src/frontend/typer/vedtaksperiode.ts

```diff
@@ -84,6 +86,18 @@ export type Utbetalingsperiode = {
     utbetaltPerMnd: number;
 };
 
+export const hentUtbetalingsperiodePåBehandlingOgPeriodeForFortsattInnvilget = (
```

Comment:
Gjelder denne bare fortsatt innvilget? Håpet var at vi kunne ha en generisk og bare håndterte null på fom/tom spesielt.

Reply:
Det vet jeg ikke helt. Den er jo ganske annerledes enn det som var, men for de andre periodetypene er kanskje fom og tom aldri null?